warehouse/query-core/src/test/java/datawave/query/testframework/AccumuloSetupHelper.java

```diff
@@ -90,7 +90,7 @@ public AccumuloSetupHelper(final Collection<DataTypeHadoopConfig> types, FileTyp
      */
     public Connector loadTables(final Logger parentLog) throws AccumuloException, AccumuloSecurityException, IOException, InterruptedException,
                     TableExistsException, TableNotFoundException, URISyntaxException {
-        return loadTables(parentLog, RebuildingScannerTestHelper.TEARDOWN.EVERY_OTHER);
+        return loadTables(parentLog, RebuildingScannerTestHelper.TEARDOWN.ALWAYS);
```

Comment:
I fear this may not catch errors that occur when the iterator is not torn down.  Basically in this mode we never call next.

Reply:
reverted